packages/design-system/src/components/combobox.tsx

```diff
@@ -203,12 +222,14 @@ export const Combobox = <Item extends BaseItem>({
         </PopperAnchor>
         {renderPopperContent({
           style: { zIndex: 1 },
+          align: "end",
+          sideOffset: 8,
```

Comment:
this seems like an adjustment for a specific use case, cc @okonet

Reply:
Yeah I'm also wondering what's this for...